images/base/Dockerfile

```diff
@@ -65,11 +91,24 @@ RUN echo "Ensuring scripts are executable ..." \
     && chmod +x /usr/local/bin/clean-install /usr/local/bin/entrypoint \
  && echo "Installing Packages ..." \
     && DEBIAN_FRONTEND=noninteractive clean-install \
-      systemd \
```

Comment:
FWIW these were previously organized into "classes" of dependency to help track what we need
- systemd is a base requirement
- conntrack etc are requirements for kubernetes / kubelet
- pigz line is for containerd
- bash etc. line is more or less for kind to use itself

(with the goal being to periodically re-evaluate the set of packages as kubernetes/containerd/... evolve).
If we're going to change this to one per line, we should at least update the comment in the source above (which outlines this)

Reply:
Ah thanks, I missed that. I'll revert